channels/pe.m3u

```diff
@@ -64,3 +64,5 @@ https://query-streamlink.lanesh4d0w.repl.co/iptv-query?streaming-ip=https://www.
 #EXTINF:-1 tvg-id="" tvg-name="" tvg-country="PE" tvg-language="" tvg-logo="https://graph.facebook.com/BestCableTvCanal3/picture?width=320&height=320" group-title="",Best Cable Perú
 #EXTVLCOPT:http-referrer=https://bestcableperu.com.pe/tv3.html
 https://live.siete.us/bestcable/bestcable/index.m3u8
```

Comment:
This link works for me even without the referer.

Reply:
Ok. I delete EXTVLCOPT line